app/controllers/api/v2/discovery_rules_controller.rb

```diff
@@ -29,9 +29,9 @@ def show
       def_param_group :discovery_rule do
         param :discovery_rule, Hash, :action_aware => true do
           param :name, String, :required => true
-          param :search, String, :required => true
-          param :hostgroup_id, Integer, :required => true
-          param :hostname, String, :required => true
+          param :search, String, :required => true, :desc => N_("query to match discovered hosts for the particular rule.")
```

Comment:
@lzap I tried to change a little the text you put in the documentation so it's shorter. Do you have a better explanation for search?